functests/utils/discovery/discovery.go

```diff
@@ -0,0 +1,12 @@
+package discovery
+
+import (
+	"os"
+	"strconv"
+)
+
+// Enabled indicates whether test discovery mode is enabled.
+func Enabled() bool {
+	discoveryMode, _ := strconv.ParseBool(os.Getenv("DISCOVERY_MODE"))
```

Comment:
I would group all env variables together, but that's only me.

Reply:
this is specific to discovery mode. what other variables are you referring to ?